express-post-json/index.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+let nextId = 1;
+const grades = {};
+const app = express();
+const newArray = [];
```

Comment:
This array should be created on-demand within your `GET` route or else it will grow and grow with each request.